src/helpers/Url/Url.test.js

```diff
@@ -350,4 +351,101 @@ describe('Url', () => {
       );
     });
   });
+  describe('flattenScales', () => {
+    it('flattenScales test from serialization', () => {
```

Comment:
```suggestion
    it('flattenScales test from the catalog', () => {
```
This is actually the catalog test